cluster/gce/trusty/configure.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
```

Comment:
Ok, I can take this as is for now and expect next PRs for handle those. Please add a TODO.